spec/workers/step_by_step_scheduled_publish_worker_spec.rb

```diff
@@ -34,12 +19,22 @@
   end
 
   it "doesn't publish a step by step scheduled in the future" do
-    step_by_step_page = create(:draft_step_by_step_page, scheduled_at: Time.now + 1.hour)
     described_class.new.perform(step_by_step_page.id)
 
     expect(StepNavPublisher).to_not have_received(:publish)
 
     step_by_step_page.reload
     expect(step_by_step_page.has_been_published?).to be false
   end
+
+  it "publishes the step by step (and generates a change note) when it's time to publish" do
```

Comment:
For readability, can you leave them as separate tests?

Also, why have they moved location? It would be easier to see what's changed if only the affected lines are updated.

Reply:
Done 👍